app/src/main/res/layout/item_add_on.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+  android:id="@+id/add_on_reward_container"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="@dimen/item_reward_width"
+  android:layout_height="wrap_content"
+  android:orientation="vertical">
+
+    <androidx.cardview.widget.CardView
+      android:id="@+id/add_on_reward_card"
+      android:layout_width="match_parent"
+      android:layout_height="300dp"
```

Comment:
I'm assuming this doesn't need to be a dimens field because it's just a placeholder for now?

Reply:
correct! I just want it to have some space to be able to see int the UI, for showing it in the gif so just QA purposes